src/basic/unit-def.h

```diff
@@ -58,6 +58,13 @@ typedef enum FreezerState {
         _FREEZER_STATE_INVALID = -EINVAL,
 } FreezerState;
 
+typedef enum UnitMarker {
+        UNIT_MARKER_NEEDS_RELOAD  = 1 << 0,
+        UNIT_MARKER_NEEDS_RESTART = 1 << 1,
```

Comment:
I'd probably define this as regular enum, and then always do (1 << UNIT_MARKER_NEEDS_RELOAD) when we need it as flag, and store the flags in an uint64_t or so

Reply:
OK. I put the flags in unsigned and added a `static_assert()` that this is enough.